Source/WebCore/dom/Document.cpp

```diff
@@ -9594,16 +9594,24 @@ void Document::updateRelevancyOfContentVisibilityElements()
 {
     if (!isObservingContentVisibilityTargets())
         return;
-    if (m_contentVisibilityDocumentState->updateRelevancyOfContentVisibilityElements(m_contentRelevancyStatusUpdate))
+    if (m_contentVisibilityDocumentState->updateRelevancyOfContentVisibilityElements(m_contentRelevancyUpdate) == DidUpdateAnyContentRelevancy::Yes)
         updateLayoutIgnorePendingStylesheets();
-    m_contentRelevancyStatusUpdate = { };
+    m_contentRelevancyUpdate = { };
 }
 
-void Document::scheduleContentRelevancyUpdate(ContentRelevancyStatus status)
+void Document::scheduleContentRelevancyUpdate(ContentRelevancy contentRelevancy)
 {
     if (!isObservingContentVisibilityTargets())
         return;
-    m_contentRelevancyStatusUpdate.add(status);
+    m_contentRelevancyUpdate.add(contentRelevancy);
```

Comment:
Should this also schedule a rendering update? The "schedule" in the name isn't quite right if it just sets a bit.

Reply:
Yes, that makes sense, added it.